src/locale/defaults.h

```diff
@@ -613,6 +616,9 @@
 #ifndef D_STR_JVC
 #define D_STR_JVC "JVC"
 #endif  // D_STR_JVC
+#ifndef D_STR_KELON
+#define D_STR_KELON "KELON"
+#endif // D_STR_KELON
```

Comment:
src/locale/defaults.h:621:  At least two spaces is best between code and comments  [whitespace/comments] [2]